actionview/lib/action_view/template.rb

```diff
@@ -1,5 +1,6 @@
 require "active_support/core_ext/object/try"
 require "active_support/core_ext/kernel/singleton_class"
+require "active_support/core_ext/module"
```

Comment:
Can we load just the delegation file? The two lines above show a precedent of being as narrow as practical.